config/routes.rb

```diff
@@ -46,15 +46,14 @@
 
   get 'new-register/thanks', to: 'new_register#thanks'
 
-  mount Spina::Engine => '/'
-
-  get '/*id' => 'pages#show', as: "page", controller: 'pages', constraints: lambda { |request|
-    !(Rails.env.development? && request.env['PATH_INFO'].starts_with?('/rails/') || request.env['PATH_INFO'].starts_with?("/#{Spina.config.backend_path}") || request.env['PATH_INFO'].starts_with?('/attachments/'))
-  }
-
-  Spina::Engine.routes.draw do
-    namespace :admin do
-      resources :registers, except: [:show]
-    end
+  namespace :admin, path: '/admin' do
+    root to: "registers#index"
+
+    resources :registers, except: [:show]
+    resources :users
+    resources :sessions
+    get "login" => "sessions#new"
```

Comment:
can we make these URLs `/login` and `/logout`? Session is a bit too much implementation detail for me.